data/json/mapgen/farm.json

```diff
@@ -129,8 +129,7 @@
         "###++### --w------w-----",
         "                        ",
         "                        ",
-        "FFFFFFFFFFFFFFFFFFFFFFFF",
-        ",,,,,,,,,,,,,,,,,,,,,,,,"
+        "FFFFFFFFFFFFFFFFFFFFFFFF"
```

Comment:
Any particular reason to delete this one instead of one of the blank rows? (Vehicle maneuvering room or something?)

Reply:
This is an alternate version of one of the OMTs above; I wanted them to match.